packages/gatsby-theme-newrelic/src/components/Callout.js

```diff
@@ -34,6 +43,10 @@ const Callout = ({ className, title, variant, children }) => {
       className={className}
       css={css`
         padding: 1.25rem;
+        margin: 1.5rem 0;
```

Comment:
This margin should be gone. It was [removed](https://github.com/newrelic/gatsby-theme-newrelic/pull/232/files#diff-dfea6acf43189bba059439ab9d704e1ca7ff214d6ed7a1d52ba176c9dbb8920dL36) in favor of adding it to the [MDX variant of this component](https://github.com/newrelic/gatsby-theme-newrelic/pull/232/files#diff-4a9b5570faaa9d3542fba8b4d8ba714c5f163bf3fa25c12a34af75e41a8a28f2)

Reply:
d'oh! doing that now.

Reply:
Thats the last thing I promise!! Thanks for doing this though. They look 🔥 🔥 🔥

Reply:
haha. NP! appreciate the 🦅 👁️